src/2048-expert/2048-expert.js

```diff
@@ -1,14 +1,15 @@
-document.addEventListener('DOMContentLoaded', ()=> {
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
`index.js` might be better as this is what Liedle and Capital use for the JavaScript file used to control the mini-game logic